lesson2/MainActivity.java/lib/src/main/java/com/example/Account.java

```diff
@@ -34,19 +21,15 @@ public long getAmount() {
 
     public void setAmount(long new_amount) {
         amount = new_amount;
-
     }
 
     public void deposit(long depositAmount) {
         amount+= depositAmount;
-
-
     }
 
     public static Account largerAccount(Account acc1, Account acc2) {
         if (acc1.getAmount() > acc2.getAmount()) {
             return acc1;
-
```

Comment:
Your code will look cleaner without so many empty lines!  Empty lines are useful if you want to visually separate two chunks of code in the same function, but they are unnecessary here before a closing brace `}`